portfolio/src/main/webapp/script.js

```diff
@@ -29,8 +29,76 @@ function addRandomGreeting() {
   factContainer.innerText = "You found a hidden fact! \n \n" + fact;
 }
 
-async function getRandomQuoteUsingAsyncAwait() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('quote-container').innerText = quote;
+// Fetches comments
+function getComments() {
+    console.log("Fetching comments")
+    const responsePromise = fetch('/data');
```

Comment:
should this be `fetch('/list-comments')`?

Reply:
That was some leftover code from one of the walk-through;  I will delete that.